packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -85,9 +85,6 @@ msgstr "Přidat stát"
 msgid "Add item"
 msgstr "Přidat položku"
 
-msgid "Add only to order confirmation page"
-msgstr "Vložit jen na stránku s potvrzením objednávky"
-
```

Comment:
Messages cannot be removed in minor version as it's considered a bc break according to our BC promise (https://docs.shopsys.com/en/9.0/contributing/backward-compatibility-promise/#translation-messages)

(this file and English messages too)

Reply:
nice point! fixed